opal/mca/hwloc/base/base.h

```diff
@@ -225,21 +180,12 @@ OPAL_DECLSPEC int opal_hwloc_base_cset2str(char *str, int len, hwloc_topology_t
 OPAL_DECLSPEC int opal_hwloc_base_cset2mapstr(char *str, int len, hwloc_topology_t topo,
```

Comment:
Above this line (I can't comment there in the Github UI) is a prototype for `opal_hwloc_base_cset2str()`.  I think this PR makes that function private / internal, and therefore it doesn't need to be prototyped here in this file, right?

Reply:
I missed this one, thanks. Removed.